qiita_db/metadata_template/base_metadata_template.py

```diff
@@ -1315,7 +1331,7 @@ def _update(self, md_template):
                     "There are no differences between the data stored in the "
                     "DB and the new data provided",
                     qdb.exceptions.QiitaDBWarning)
-                return
+                return set([]), set([])
```

Comment:
Is it better to return two empty sets vs None, or (None, None), throw an Exception, etc?